config/testdata/status-defaults/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta1
```

Comment:
A Flux Kustomization file name should never be kustomization.yaml, as this file name is reserved for Kustomize configs. How come this manifests can be applied on the cluster? It has no spec and it should fail validation: https://github.com/fluxcd/kustomize-controller/blob/main/config/crd/bases/kustomize.toolkit.fluxcd.io_kustomizations.yaml#L327